labgrid/remote/client.py

```diff
@@ -41,6 +42,10 @@ class UserError(Error):
 class ServerError(Error):
     pass
 
+WRITE_IMAGE_TOOLS = {
+    "dd": WriteMode.DD,
+    "bmaptool": WriteMode.BMAPTOOL,
+}
```

Comment:
```suggestion
```
Wouldn't it be better to convert the Mode enum members into these strings?
In `labgrid/driver/usbstoragedriver.py`:
```python
class Mode(enum.Enum):
    DD = "dd"
    BMAPTOOL = "bmaptool"
```